code/mob/living/silicon.dm

```diff
@@ -28,6 +28,9 @@ ADMIN_INTERACT_PROCS(/mob/living/silicon, proc/pick_law_rack)
 
 	var/static/regex/monospace_say_regex = new(@"`([^`]+)`", "g")
 
+	var/obj/minimap_controller/alertmap_controller
+	var/atom/movable/minimap_ui_handler/minimap_controller/general_alert/alert_minimap_ui
```

Comment:
Is there a reason we're making a new UI for every silicon? It might be cleaner to just have these both be static variables.

Reply:
My understanding is that if they share the controller/ui, then the map zoom/pan would be shared across all silicons. I skip initialization until they open the alert map, but I'm not sure there's a better way.